fastlane/lib/fastlane/actions/xcodebuild.rb

```diff
@@ -108,6 +111,10 @@ def self.run(params)
             raw_buildlog = params[:raw_buildlog]
           end
 
+          if params.key? :xcpretty_utf
+            xcpretty_utf = params[:xcpretty_utf]
```

Comment:
Can't we just combine that with https://github.com/fastlane/fastlane/pull/7659/files#diff-ce8c49563f8022fccd8ec5739763a461R102, and have it in one line?